### ISSUE TITLE
Unexpected character encountered while parsing value: M. Path ''

### ISSUE THREAD

**Issue by deepforest** [lukencode/FluentEmail#88]
Hi,
Trying to send an HTML body email, using Mailgun (no api-key and domain specified), I've got the following exception:

> Unexpected character encountered while parsing value: M. Path ''

Digging into QuickResponse<T>.FromMessage method, I've found that:
```
if (message.IsSuccessStatusCode)
{
    response.Data = JsonConvert.DeserializeObject<T>(response.ResponseBody);
}
```
 throws the exception when trying to convert the response 'response.ResponseBody' to json object.
The value of 'response.ResponseBody' by the time of debugging was: "**Mailgun Magnificent API**".

I know that this is dumb API response, and we expect to get valid error response, but unfortunately, someone has to deal with stupidity ;)

Great API, Thanks!

**Comment by bjcull** [comment]
Hey, good call. Fixed with https://github.com/lukencode/FluentEmail/commit/38e1c741fb7bd27faa1b3e613a78134ed609bcba

**Comment by bjcull** [comment]
Released in 2.4.0
